docs/integrations/notification/index.rst

```diff
@@ -0,0 +1,126 @@
+#####################################
+ Monitor Experiment Through Webhooks
+#####################################
+
+This section includes user guides for how to set up notifications for experiments.
+
+Determined now supports webhooks for sending notification to monitor experiment state change.
+
+**********
+ Security
+**********
+
+Determined utilizes ``signed_payload`` and ``timestamp`` headers. You will need to configure a
```

Comment:
We should note what these are actually used for: to verify that webhook calls from Determined are genuine.